synapse/tests/test_lib_view.py

```diff
@@ -191,12 +190,6 @@ async def test_view_trigger(self):
             nodes = await alist(core.eval('[ test:int=11 ]', opts={'view': view2.iden}))
             self.len(1, nodes)
 
-            nodes = await alist(view2.eval('test:str'))
```

Comment:
Do we have a test which shows that the parents trigger does not fire until AFTER we have performed the merge?

Reply:
we do now